packages/gestalt/src/RadioButton.js

```diff
@@ -53,43 +58,65 @@ export default class RadioButton extends React.Component<Props, State> {
   handleFocus = () => this.setState({ focused: true });
 
   render() {
-    const { checked, disabled, id, name, size, value } = this.props;
+    const { checked, disabled, id, label, name, size, value } = this.props;
     const { focused } = this.state;
+
     return (
-      <div
-        className={classnames(styles.RadioButton, {
-          [styles.RadioButtonIsFocused]: focused,
-          [styles.RadioButtonSm]: size === 'sm',
-          [styles.RadioButtonMd]: size === 'md',
-          [styles.RadioButtonWhiteBg]: !disabled || checked,
-          [styles.RadioButtonLightGrayBg]: disabled && !checked,
-        })}
+      <Box
+        alignItems="center"
+        display="flex"
+        justifyContent="start"
+        marginLeft={-1}
+        marginRight={-1}
       >
-        <input
-          checked={checked}
-          className={classnames(styles.Input, {
-            [styles.InputEnabled]: !disabled,
-            [styles.InputSm]: size === 'sm',
-            [styles.InputMd]: size === 'md',
-          })}
-          disabled={disabled}
-          id={id}
-          name={name}
-          onBlur={this.handleBlur}
-          onChange={this.handleChange}
-          onFocus={this.handleFocus}
-          type="radio"
-          value={value}
-        />
-        {checked && (
```

Comment:
@AlbertCarreras looks like the `checked` state is broken for RadioButton: 
![image](https://user-images.githubusercontent.com/127199/77326259-c741c980-6cd6-11ea-9712-b79a5042446a.png)

Could you also add a test to ensure the `check` / `dot` shows up when `checked` is set to `true`?

Reply:
DONE